src/plugins/auto/plugin.cpp

```diff
@@ -314,7 +314,8 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     if (modelPath.empty() && network.getFunction() == nullptr) {
         IE_THROW() << GetName() << " device supports just ngraph network representation";
     }
-
+    // to use plugin's name as the log tag
+    _LogTag = GetName();
```

Comment:
is possiable to GetName() in MultiDeviceInferencePlugin::MultiDeviceInferencePlugin()